specs/FamilySafety.md

```diff
@@ -0,0 +1,64 @@
+Family Safety
+===
+
+# Background
+Provide end evelolper a new API to toggle Family Safety feature on and off. Once Family Safety is enabled, developer won't be able to turn it off while webview2 instance is running. Once enable, it will provide the same functionally as the browser like: Activity report, Safe Search and Web Filtering. Please see https://www.microsoft.com/en-us/microsoft-365/family-safety for details on Family Safety. 
+
+# Examples
+## WinRT and .NET   
+```c#
+void WebView_ToggleFamilySafetyFeature(object sender, object e)
+{
+    Control.CreationProperties.EnabledFamilySafety = true;
+}
+
+
+```
+## Win32 C++
+```cpp
+// Enable the Family Safety feature upon webview environment creation complete
+HRESULT ToggleFamilySafety()
+{
+    Microsoft::WRL::ComPtr<ICoreWebView2EnvironmentOptions3> optionsStaging3;
+    if (options.As(&optionsStaging3) == S_OK)
+    {
+        optionsStaging3->put_EnabledFamilySafety(TRUE);
```

Comment:
This sample code is incomplete. You have options.As(...) but options is never defined. It would be good to show creating the options, setting your new property, and calling to create the environment.